index.html

```diff
@@ -52,14 +52,14 @@ <h2 class="project-tagline">QUIC support for Go</h2>
 
 <p>QUIC aims to solve the head-of-line blocking issue and the TLS connection round-trip problem. A detailed project description is available in the <a href="http://www.chromium.org/quic">QUIC project page</a>.</p>
 
-<p>Unfortunately, since QUIC is a experimental protocol, there's only one implementation of the protocol. And that implementation exists in project Chromium. Google provides toy server/client to play with. But the source code itself is not designed to be a library. It is heavily coupled to the existing Chromium sources.</p>
+<p>Unfortunately, since QUIC is an experimental protocol, there's only one client-side implementation of the protocol. And that implementation exists in project Chromium. Google provides toy server/client to play with. But the source code itself is not designed to be a library. It is heavily coupled to the existing Chromium sources.</p>
```

Comment:
As far as I know, This is also true for server-side implementation, not only for client-side.